core/kazoo_bindings/src/kazoo_bindings.erl

```diff
@@ -819,13 +818,14 @@ bindings(Routing) ->
                              }=Bind, Acc) ->
                       case Binding =:= Routing
                           orelse matches(BParts, RoutingParts)
-                          orelse matches(RoutingParts, BParts)
```

Comment:
@fenollp this one should not be removed. `bindings` should look both ways. `candidates` and `get_candidates` should behave before our changes.